### DISCUSSION TITLE
How to achieve typed parameters for yaml files

### DISCUSSION THREAD

**Discussion by ngoc-quoc-huynh** [slang-i18n/slang#260] [Q&A] [answered]
Currently this yaml file:
```yaml
option(int index): 'Option $index'
````

Generates this dart output.
```dart
({required Object index}) => 'Option ${index}'
```

How can i achieve typed parameters like in the [README](https://pub.dev/packages/slang#-typed-parameters) but for yaml files?

**Comment by Tienisto** [comment, marked as answer]
Can you try `option: 'Option ${index: int}'`

**Reply by ngoc-quoc-huynh** [reply]
Yeah this works. Somehow this didn't worked earlier. Maybe I did something wrong. Thanks for your help!
